mod-erm-usage-server/src/test/java/org/folio/rest/impl2/UsageDataProvidersIT.java

```diff
@@ -265,46 +268,38 @@ public void checkThatWeCanAddGetPutAndDeleteUsageDataProviders() {
 
   @Test
   public void checkThatWeCanSearchByCQL() {
-    String mockedOkapiUrl = "http://localhost:" + wireMockRule.port();
+    var udproviders = List.of(udprovider, udprovider2);
 
-    // POST provider without aggregator
-    UsageDataProvider udp =
-        given()
-            .body(Json.encode(udprovider))
-            .header("X-Okapi-Tenant", TENANT)
-            .header("X-Okapi-Url", mockedOkapiUrl)
-            .header("content-type", APPLICATION_JSON)
-            .header("accept", APPLICATION_JSON)
-            .request()
-            .post(BASE_URI)
-            .thenReturn()
-            .as(UsageDataProvider.class);
-    assertThat(udp.getLabel()).isEqualTo(udprovider.getLabel());
-    assertThat(udp.getId()).isNotEmpty();
+    // POST two providers
+    udproviders.forEach(this::postUdp);
 
-    String cqlLabel = "?query=(label=\"" + udprovider.getLabel() + "\")";
-    UsageDataProviders queryResult =
-        given()
-            .header("X-Okapi-Tenant", TENANT)
-            .header("content-type", APPLICATION_JSON)
-            .header("accept", APPLICATION_JSON)
-            .when()
-            .get(BASE_URI + cqlLabel)
-            .thenReturn()
-            .as(UsageDataProviders.class);
-    assertThat(queryResult.getUsageDataProviders().size()).isEqualTo(1);
-    assertThat(queryResult.getUsageDataProviders().get(0).getLabel()).isEqualTo(udp.getLabel());
-    assertThat(queryResult.getUsageDataProviders().get(0).getId()).isEqualTo(udp.getId());
+    // GET by CQL: search for label
+    get("""
+            label="%s"
+            """.formatted(udprovider.getLabel()))
```

Comment:
Thank you for the addition. Would you mind using string literals instead of text block syntax?